scripts/pkg/update-docs_test.go

```diff
@@ -0,0 +1,89 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package pkg
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const version = "v1.11.0"
+
+func Test_translateRelativeLinks(t *testing.T) {
+	tests := []struct {
+		name     string
+		fullPath string
+		content  []byte
+		expected []byte
+	}{
+		{
+			name:     "no translation-absolute external link",
+			fullPath: "website/content/docs/v1.11.0/somefile.md",
+			content:  []byte("texts[filename](https://github.com/antrea-io/antrea/blob/main/test/test.yaml)texts"),
+			expected: []byte("texts[filename](https://github.com/antrea-io/antrea/blob/main/test/test.yaml)texts"),
+		},
+		{
+			name:     "no translation-relative link to a file which is synced",
```

Comment:
I am not sure what "synced" means. I assume it means "translated into a website doc page"? Since you use "synced" to describe multiple tests, maybe add a comment to define the term, just above the `tests` list definition.

Reply:
Here, 'synced' means that the file is present in website repository as it has been copied(synced) from Antrea repository by `func syncDirs` while updating docs.

Reply:
please add a comment to this effect